gems/smithy/lib/smithy/polishes.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Smithy
+  # @api private
+  module Polishes
```

Comment:
Naming discussion.  `Polishes` reads a little weirdly to me - it almost feels like this should be `Polishers`.  Ie, a `Polisher` can take the `polish` action, eg: `@polishers.each { |p| p.polish }`.

Reply:
Polishes is the plural of polish. We can think about changing this.